localgov_directories.module

```diff
@@ -444,3 +444,12 @@ function localgov_directories_remove_block_from_content_type_sidebar(string $blo
 
   return TRUE;
 }
+
+/**
+ * Implements hook_form_alter().
+ */
+function localgov_directories_form_alter(&$form, &$form_state, $form_id) {
+  if ($form_id === 'node_localgov_directories_venue_form') {
```

Comment:
Should this go into the [localgov_directories_venue](https://github.com/localgovdrupal/localgov_directories/tree/2.x/modules/localgov_directories_venue) submodule?

Reply:
Yes, that might make more sense.